catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1543,21 +1550,35 @@ protected String handleWebApplicationException(WebApplicationException wae) {
         // meaningful since it will not show the root cause of the exception
         // because the ExceptionReport was sent from CSW as an "OK" JAX-RS
         // status rather than an error status.
-        String msg = "Error received from CSW Server " + cswSourceConfiguration.getId() + "\n"
+        String msg = CSW_SERVER_ERROR + " " + cswSourceConfiguration.getId() + "\n"
                 + cswException.getMessage();
-        LOGGER.error(msg, wae);
+        LOGGER.warn(msg, wae.getMessage());
 
         return msg;
     }
 
-    private String handleClientException(Exception ce) {
+    protected String handleClientException(Exception ce) {
```

Comment:
Changed to protected so it can be used by other CSW sources that inherit from AbstractCswSource.